store/modules/ui.js

```diff
@@ -51,6 +52,7 @@ const state = {
   txDialogActionRoute: null,
   txDialogActionText: '',
   isShowingPromptNotificationDialog: false,
+  likeCoinUsdPrice: null,
```

Comment:
is `likeCoinUsdPrice` a string? should it be `''` instead
also can the `action` / `getter` be better named to reflect it is a string, instead of number/bignumber?